src/eiffel_graphql_api/storage.py

```diff
@@ -85,6 +97,11 @@ def main(args):
     args = parse_args(args)
     setup_logging(args.loglevel)
 
+    # For some reason the SIGTERM signal gets lots when consuming from RabbitMQ ,
```

Comment:
lots -> lost

Reply:
Oops. Done. Also removed the extra space after "RabbitMQ".